### DISCUSSION TITLE
responsive design

### DISCUSSION THREAD

**Discussion by sheraz-saqib** [tailwindlabs/tailwindcss#15042] [Kind words]
max-[600px]:bg-sky-300
 max-[custom_units]:w-full
 max-2xl:h-full
 
 these are media qureys classes not working

**Comment by wongjn** [comment]
Consider checking whether your Tailwind configuration conforms [to the restrictions for `max-*` variants to work](https://github.com/tailwindlabs/tailwindcss/pull/9558#user-content-restrictions). Otherwise, consider providing a reproducible example please.

**Comment by ayakhaled0001** [comment]
### Tailwind Version Compatibility
Custom screen sizes like max-[600px] and arbitrary values were introduced in Tailwind CSS v3.2+. Check your version:

bash
`npm list tailwindcss`
If you're on an older version, update:

bash
`npm install tailwindcss@latest`
